Modules/Scripted/Home/Home.py

```diff
@@ -0,0 +1,171 @@
+import qt
+
+import slicer
+from slicer.ScriptedLoadableModule import (
+    ScriptedLoadableModule,
+    ScriptedLoadableModuleLogic,
+    ScriptedLoadableModuleWidget,
+)
+from slicer.util import VTKObservationMixin
+
+import SlicerCustomAppUtilities
+
+# Import to ensure the files are available through the Qt resource system
+from Resources import HomeResources
+
+
+class Home(ScriptedLoadableModule):
+    """The home module allows to orchestrate and style the overall application workflow.
+
+    It is a "special" module in the sense that its role is to customize the application and
+    coordinate a workflow between other "regular" modules.
+
+    Associated widget and logic are not intended to be initialized multiple times.
+    """
+
+    def __init__(self, parent):
+        ScriptedLoadableModule.__init__(self, parent)
+        self.parent.title = "Home"
+        self.parent.categories = [""]
+        self.parent.dependencies = []
+        self.parent.contributors = ["Sam Horvath (Kitware Inc.)", "Jean-Christophe Fillion-Robin (Kitware Inc.)"]
+        self.parent.helpText = """This module orchestrates and styles the overall application workflow."""
+        self.parent.helpText += self.getDefaultModuleDocumentationLink()
+        self.parent.acknowledgementText = """..."""  # replace with organization, grant and thanks.
+
+        # Create the Home module's widget representation just to ensure that HomeWidget.setup
+        # gets called and the styling and UI modifications are run.
+        def touch_widget_representation():
+                slicer.modules.home.widgetRepresentation()
+        slicer.app.connect("startupCompleted()", touch_widget_representation)
```

Comment:
(@sjh26 please see this especially -- this is how I ended up getting your suggestion to work.)

Reply:
lgtm, seems to work fine, and is a nice way to make sure the entire widget/logic is set up